.travis.yml

```diff
@@ -26,9 +25,11 @@ script:
   - molecule verify -s $SCENARIO
   - cd $TRAVIS_BUILD_DIR
 after_success:
-  - docker commit `docker ps |grep "centos:7" |awk {'print $1'}` quay.io/egi/ui
+  - docker commit ui-centos7 quay.io/egi/ui:centos7
```

Comment:
What about https://hub.docker.com/u/egifoundation/?

Reply:
Indeed it would be nice to be able to push the image to dockerhub too.

Reply:
I vote against docker hub: 

1) it's a repository, like quay, so nothing of value is lost by using quay.
2) quay is a product in the RedHat family, with long-term support
3) most importantly, we get Clair for free, and there is actually feedback in the code from the vulnerabilities raised by it. Does docker hub do container scanning?
4) It's a bit more closed off than other registries, in "Docker World :tm: " and I feel like we're promoting some form of antisocial behaviour (I'm totally not standing behind this statement as a policy, just a feeling I have personally)

Sure , _also_ pushing to docker hub isn't a huge PITA, it's a few extra lines here, but I just don't see the point. 

What's the motivation behind using Docker Hub?

Reply:
I'm fine with not using docker hub, but until now, that was the source for containers from EGI. 

We just need to be clear on where do we publish.

BTW, for me it's not clear the price of quay? is it free for public stuff?

Reply:
>BTW, for me it's not clear the price of quay? is it free for public stuff?

It's got a similar pricing model to hub - free tier, paid for private, etc.

Reply:
> We just need to be clear on where do we publish.

I totally agree. We don't have to have a restrictive policy (MUST), but we can have a images SHOULD be pushed to hub and quay ?